src/main/java/com/revature/ams/Booking/dtos/BookingRequestDTO.java

```diff
@@ -2,6 +2,10 @@
 
 /**
  * TODO: DOCUMENT ME
+ * This class declares data and implements 2 constructors which initializing data currently not being used in the
```

Comment:
I would suggest mentioning that this is a Data Transfer Object or an object in general with [list attributes here]